python/its_client/quadtree.py

```diff
@@ -59,6 +66,48 @@ def get_neighbour(quadtree, direction):
     return result[::-1]
 
 
+def get_neighborhood(quadkey):
+    """
+    Returns surrounding quadkeys list
+
+    :param quadkey:     Quadkey to get the neighborhood of
+    :return:            a list containing the quadkeys next to the one provided
+    """
+    neighbors = []
+
+    quadkey = unslash(quadkey)
+
+    up = get_neighbour(quadkey, "up")
+    down = get_neighbour(quadkey, "down")
+
+    neighbors.append(get_neighbour(up, "left"))
+    neighbors.append(up)
+    neighbors.append(get_neighbour(up, "right"))
+    neighbors.append(get_neighbour(quadkey, "left"))
+    neighbors.append(get_neighbour(quadkey, "right"))
+    neighbors.append(get_neighbour(down, "left"))
+    neighbors.append(down)
+    neighbors.append(get_neighbour(down, "right"))
+
+    return neighbors
+
+
+def slash(unslashed_quadkey: str) -> str:
+    if UNSLASHED_KEY_RE.match(unslashed_quadkey):
```

Comment:
hum, it's well known re are very expensive in Python, to avoid : no way to use un "simple" str "in" instead? Or another solution? Correct for now, but could be too slow further...

Reply:
I removed the `re` usage in favor of less complex str inspection methods